lib/3scale/backend/storage.rb

```diff
@@ -226,8 +228,8 @@ def sentinel_to_hash(sentinel)
             valid_uri_str = to_redis_uri(sentinel)
             # it is safe to perform URI parsing now
             uri = URI.parse valid_uri_str
-
-            { host: uri.host, port: uri.port }
+            sentinel_port = uri.port || DEFAULT_SENTINEL_PORT
```

Comment:
sentinels are iterated. Instead of applying default case in configuration format, we could iterate again checking port exists, otherwise, default port. Double iteration, single place of checking for default sentinel port.